sdk/metricsadvisor/perf-tests/ai-metrics-advisor/test/listIncidents.spec.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { PerfStressOptionDictionary, getEnvVar } from "@azure/test-utils-perfstress";
+import { MetricsAdvisorTest } from "./metricsAdvisor.spec";
+interface MetricsAdvisorTestOptions {}
+
+export class IncidentsListTest extends MetricsAdvisorTest<MetricsAdvisorTestOptions> {
+  alertId: string;
+  alertConfigId: string;
+  public options: PerfStressOptionDictionary<MetricsAdvisorTestOptions> = {};
+  constructor() {
+    super();
+    this.alertId = getEnvVar("METRICS_ADVISOR_ALERT_ID");
+    this.alertConfigId = getEnvVar("METRICS_ADVISOR_ALERT_CONFIG_ID");
+  }
+
+  async runAsync(): Promise<void> {
+    let iter = this.client.listIncidents({
```

Comment:
If possible, let's use code more similar to our sample:

https://github.com/Azure/azure-sdk-for-js/blob/e59290ba59b1239488a91cbead024970688b3651/sdk/metricsadvisor/ai-metrics-advisor/samples/typescript/src/incidentsAndAlerts.ts#L209-L210

Reply:
done